li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/largemessage/MessageAssembler.java

```diff
@@ -26,6 +27,18 @@
    */
   AssembleResult assemble(TopicPartition tp, long offset, byte[] segmentBytes);
```

Comment:
Can you add a `TODO` here explaining that we should mark this interface as deprecated for the next major version bump?

Reply:
Revised.